### PR TITLE
Modefied the Quickstart page of Orca

### PR DESCRIPTION
**Pull request by leonardozcm**

### CODE REVIEW

File: docs/docs/Orca/orca-pytorch-quickstart.md [hunk 1]
```diff
@@ -34,8 +34,9 @@ sc = init_orca_context(
         "spark.task.maxFailures": "1",
         "spark.driver.extraJavaOptions": "-Dbigdl.failure.retryTimes=1"})
 ```
-**Note:** you should `export HADOOP_CONF_DIR=/path/to/hadoop/conf/dir`
-* Reference: [Orca Context](https://analytics-zoo.github.io/master/#Orca/context/)
+**Note:** You should `export HADOOP_CONF_DIR=/path/to/hadoop/conf/dir`
```
Comment (hkvision) [inline docs/docs/Orca/orca-pytorch-quickstart.md:37]: missing `.` at the end

File: docs/docs/Orca/orca-pytorch-quickstart.md [hunk 2]
```diff
@@ -64,42 +65,46 @@ class LeNet(nn.Module):
...
+adam = Adam(1e-4)
 ```
 
-### **Step 3: Fit with Orca PyTorch Estimator**
-1. Define the data in whatever way you want. Orca just needs a dataloader, a callable datacreator or an Orca SparkXShards
-```python
-torch.manual_seed(args.seed)
+### **Step 3: Fit with Orca PyTorch Estimator**  
 
+1)  Define the data in whatever way you want. Orca just needs a dataloader, a callable datacreator or an [Orca SparkXShards](../../#Orca/data/).
```
Comment (hkvision) [inline docs/docs/Orca/orca-pytorch-quickstart.md:73]: a Pytorch DataLoader (link), a data creator function or ...

File: docs/docs/Orca/orca-pytorch-quickstart.md [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+
+* Well-organized and flexible
 * Easier to reproduce
-* Utilize distributed training without changing your model
+* Able to perform distributed training without changing your model
 
 ### **Step 0: Prepare environment**
-We recommend you to use [Anaconda](https://www.anaconda.com/distribution/#linux) to prepare the environments, especially if you want to run on a yarn cluster(yarn-client mode only).
-**Note:** You can install the latest analytics whl by following instructions ([here](https://analytics-zoo.github.io/master/#PythonUserGuide/install/#install-the-latest-nightly-build-wheels-for-pip)).
+We recommend you to use [Anaconda](https://www.anaconda.com/distribution/#linux) to prepare the environments, especially if you want to run on a yarn cluster(yarn-client mode only).
```
Comment (hkvision) [inline docs/docs/Orca/orca-pytorch-quickstart.md:11]: space before (

File: docs/docs/Orca/orca-pytorch-quickstart.md [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
+* Well-organized and flexible
 * Easier to reproduce
-* Utilize distributed training without changing your model
+* Able to perform distributed training without changing your model
 
 ### **Step 0: Prepare environment**
-We recommend you to use [Anaconda](https://www.anaconda.com/distribution/#linux) to prepare the environments, especially if you want to run on a yarn cluster(yarn-client mode only).
-**Note:** You can install the latest analytics whl by following instructions ([here](https://analytics-zoo.github.io/master/#PythonUserGuide/install/#install-the-latest-nightly-build-wheels-for-pip)).
+We recommend you to use [Anaconda](https://www.anaconda.com/distribution/#linux) to prepare the environments, especially if you want to run on a yarn cluster(yarn-client mode only).  
+**Note:** You can install the latest analytics-zoo whl by following [instructions](../../#PythonUserGuide/install/#install-the-latest-nightly-build-wheels-for-pip).
```
Comment (hkvision) [inline docs/docs/Orca/orca-pytorch-quickstart.md:12]: following the instructions [here](link)

File: docs/docs/Orca/orca-tf-quickstart.md [hunk 5]
```diff
@@ -87,8 +87,8 @@ acc = accuracy(logits, labels)
 ```
 
 ### **Step 3: Fit with Orca TensorFlow Estimator**
-1. Define the dataset in whatever way you want. Orca just needs tf.data.Dataset, Spark DataFrame or Orca SparkXShards.
-```python
+1)  Define the dataset in whatever way you want. Orca supports [tf.data.Dataset](https://www.tensorflow.org/api_docs/python/tf/data/Dataset), [Spark DataFrame](https://spark.apache.org/docs/latest/sql-programming-guide.html) and [Orca SparkXShards](../../#Orca/data/).
+```pythoin
```
Comment (hkvision) [inline docs/docs/Orca/orca-tf-quickstart.md:91]: typo

File: docs/docs/Orca/orca-tf-quickstart.md [hunk 6]
```diff
@@ -1,14 +1,14 @@
...
+Scaling your TensorFlow applications with Orca makes your code:
+
+* Well-organized and flexible
 * Easier to reproduce
-* Utilize distributed training without changing your model
+* Able to perform distributed training without changing your model
 
 ### **Step 0: Prepare environment**
-Download and install latest analytics whl by following instructions ([here](https://analytics-zoo.github.io/master/#PythonUserGuide/install/#install-the-latest-nightly-build-wheels-for-pip)).
+Download and install latest analytics-zoo whl by following [instructions](../../#PythonUserGuide/install/#install-the-latest-nightly-build-wheels-for-pip).
```
Comment (hkvision) [inline docs/docs/Orca/orca-tf-quickstart.md:11]: the ... here
and the link is invalid (remove #)

File: docs/docs/Orca/orca-tf-quickstart.md [hunk 7]
```diff
@@ -32,11 +32,11 @@ init_orca_context(cluster_mode="local", cores=4)
 # run in yarn client mode
 init_orca_context(cluster_mode="yarn-client", num_nodes=2, cores=2, driver_memory="6g")
 ```
-* Reference: [Orca Context](https://analytics-zoo.github.io/master/#Orca/context/)
+View [Orca Context](../../#Orca/context/) for more details.
```
Comment (hkvision) [inline docs/docs/Orca/orca-tf-quickstart.md:35]: the link is invalid

**Review by hkvision — APPROVED** [review]
